iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -127,14 +133,23 @@ public void commitTask(TaskAttemptContext originalContext) throws IOException {
           .run(output -> {
             Table table = HiveIcebergStorageHandler.table(context.getJobConf(), output);
             if (table != null) {
-              HiveIcebergWriter writer = writers.get(output);
+              Collection<DataFile> dataFiles = Lists.newArrayList();
+              Collection<DeleteFile> deleteFiles = Lists.newArrayList();
               String fileForCommitLocation = generateFileForCommitLocation(table.location(), jobConf,
```

Comment:
Is this used?

Reply:
This was used when the sorting of the delete branch was missing. Since sorting is a must we never generate a plan any more which can handle both branches in the mapper and commit files are generated by separate jobs so I reverted this change.